### ISSUE TITLE
`pulumi version` outputs a newline and nothing else

### ISSUE THREAD

**Issue by AaronFriel** [pulumi/pulumi#5490]
The command `pulumi version` isn't printing out a version.

## Expected Behavior

```sh
> pulumi version
$SOME_VERSION_HERE
```

## Current Behavior

```sh                      .
root@AARONFRIEL-DESKTOP:~# pulumi version

root@AARONFRIEL-DESKTOP:~# pulumi version | xxd
00000000: 0a                           
```

Prints exactly a newline.

## Steps to Reproduce

The following shell script downloads and installs Pulumi from tarball, and should be idempotent once Pulumi is installed. If the version installed matches, it does not reinstall it, and instead returns.

Because `pulumi version` does not print the version, it is not idempotent. It always downloads the current version of Pulumi.

```bash
PULUMI_VERSION="2.11.0"
PULUMI_BIN_LOCATION="$HOME/.pulumi/bin"

if ! command -v pulumi > /dev/null || ! (pulumi version 2>/dev/null | grep $PULUMI_VERSION > /dev/null); then
  PULUMI_BIN_LOCATION="$HOME/.pulumi/bin"
  mkdir -p "$PULUMI_BIN_LOCATION"
  bin_dir="$(readlink -f "$PULUMI_BIN_LOCATION")"

  tmpdir=$(mktemp -d)
  curl "https://get.pulumi.com/releases/sdk/pulumi-v${PULUMI_VERSION}-linux-x64.tar.gz" | tar zx -C "$tmpdir"
  cp -r "$tmpdir/." "$bin_dir"
  rm -rf "$tmpdir"
  ln -s "$(readlink -f "${bin_dir}/pulumi")" /usr/local/bin/pulumi
  PATH="$PATH:$bin_dir"
  export PATH
fi
```

**Comment by jaxxstorm** [comment]
@AaronFriel im working on fixing this, the v2.11.0 release didn't work as expected and the binary is broken.

I'll be pushing v2.11.1 shortly to fix it, apologies for this.

**Comment by AaronFriel** [comment]
Nice, thanks!

**Comment by stack72** [comment]
Hi @AaronFriel 

Apologies for this, this has been fixed with the v2.11.1 release:

```
▶ pulumi version
v2.11.1
```
